examples/share/README.md

```diff
@@ -0,0 +1,52 @@
+Tests for Lando Share Command
+=============================
+
+This example provides tests for the `lando share` command.
+
+Getting Started
+---------------
+
+This example is here for functional testing only.
+
+Helpful Commands
+----------------
+
+```bash
+# Get app info
+lando info
+
+# Verify the lando share command works
+lando share --url http://localhost:8081
+lando ssh -c "curl -I https://share.localtunnel.me |grep \'200 OK\'"
```

Comment:
This is going to be quite brittle, you can get a different domain assigned from localtunnel depending on what else is using it. parallel test runs or any other person using that domain might collide

Reply:
i dont think we have an option to pass in a custom domain to localtunnel so either we need to 

1. Implement that option as part of this (https://github.com/localtunnel/localtunnel#opts)
2. Change the app name from share to something sufficiently obscure so as to reduce the brittleness

Reply:
@serundeputy looks like there is already a card for the custom domain option: lando/lando#1145